source/frontend/src/utils/mapPropertyUtils.ts

```diff
@@ -172,6 +172,30 @@ export const getFeatureBoundedCenter = (feature: Feature<Geometry, GeoJsonProper
   }
 };
 
+export function apiToMapProperty(fileProperty: ApiGen_Concepts_FileProperty): IMapProperty {
+  return {
+    propertyId: fileProperty.propertyId,
+    pid: fileProperty.property?.pid?.toString(),
+    pin: fileProperty.property?.pin?.toString(),
+    latitude: fileProperty.property?.location?.coordinate.y,
+    longitude: fileProperty.property?.location?.coordinate.x,
+    fileLocation: {
+      lat: fileProperty.property?.location?.coordinate.y,
+      lng: fileProperty.property?.location?.coordinate.x,
+    },
```

Comment:
just want to double check - I believe existing logic for "fileLocation" is to have the potentially different lat/long for a property in a file (regardless of original lat/long at property level). As such Concept_FileProperty has its own **location** property. 

I think the IMapProperty mapLocation should be assigned the **fileProperty.location** (instead of **fileProperty.property?.location**)

Could you please check with current implementation?